datacube_alchemist/upload.py

```diff
@@ -1,10 +1,17 @@
 import os
 import tempfile
+import boto3
 from distutils.dir_util import copy_tree
 from pathlib import Path
+import mimetypes
+try:
+    from urlparse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
```

Comment:
this is python2 work-around, we don't do python2

Reply:
Thanks, I'll remove it